<!DOCTYPE html>
<html lang="en-us">

<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Source Themes Academic 4.7.0">

  

  
  
  
  
  
    
    
    
  
  

  <meta name="author" content="Sebastiaan Vanuytven">

  
  
  
    
  
  <meta name="description" content="Single-cell sequencing techniques allow the study of the subclonal architecture of tumours and reveal the co-occurrence of (driver) mutations as well as their order of acquisition over molecular pseudo-time. The development of multi-omics techniques such as genome-and-transcriptome sequencing (G&amp;T-seq) will enable us to study the diversity of cancer cell states that arises within a tumour, at its most fundamental level, the cell. Recently we developed Gtag&amp;T-seq, an improved G&amp;T-seq method that applies direct genome tagmentation (Gtag), thereby avoiding upfront whole-genome amplification (WGA) and its associated cost and biases. We applied both methodologies to a human melanoma PDX melanoma model before treatment with BRAF and MEK inhibition, and at minimal residual disease. Three genomic subclones were observed at both timepoints, however their relative abundancy at cellular level changes upon drug exposure. Using the RNA of the same single cells, we showed that transcriptome-based DNA copy number inference has limited resolution and accuracy, underlining the importance of affordable multi-omic approaches. In addition, we discerned 10 focal amplifications on chromosomes 13 and 22 of which only genes of the chr22q11.21 focal amplification showed clear gene dosage effects. Using the Gtag&amp;T data, we were able to determine breakpoints at near base-pair resolution, which is not possible with conventional G&amp;T-seq. Finally, we mapped known melanoma resistant cell states on top of the phylogenetic cell lineage tree revealing that some cell states do not occur in all of the subclones. In conclusion, Gtag&amp;T-seq is a novel, low-cost, and accurate single-cell multi-omics method that enables the exploration of somatic genetic alterations and their functional consequences in single cells at scale.">

  
  <link rel="alternate" hreflang="en-us" href="https://svanuytven.github.io/talk/biomina2022/">

  


  
  
  
  <meta name="theme-color" content="#2962ff">
  

  
  

  
  
  
  
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.0-1/css/all.min.css" integrity="sha256-4w9DunooKSr3MFXHXWyFER38WmPdm361bQS/2KUWZbU=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css" integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin="anonymous">

    
    
    
      
    
    
      
      
        
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.10/styles/github.min.css" crossorigin="anonymous" title="hl-light">
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.10/styles/dracula.min.css" crossorigin="anonymous" title="hl-dark" disabled>
        
      
    

    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.5.1/leaflet.css" integrity="sha256-SHMGCYmST46SoyGgo4YR/9AlK1vf3ff84Aq9yK4hdqM=" crossorigin="anonymous">
    

    

    
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
        <script src="https://cdnjs.cloudflare.com/ajax/libs/lazysizes/5.1.2/lazysizes.min.js" integrity="sha256-Md1qLToewPeKjfAHU1zyPwOutccPAm5tahnaw7Osw0A=" crossorigin="anonymous" async></script>
      
    
      

      
      

      
    
      

      
      

      
    
      

      
      
        
      

      
    
      

      
      

      
    

  

  
  
  
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat:400,700%7CRoboto:400,400italic,700%7CRoboto+Mono&display=swap">
  

  
  
  
  
  <link rel="stylesheet" href="/css/academic.css">

  




  


  

  <link rel="manifest" href="/index.webmanifest">
  <link rel="icon" type="image/png" href="/images/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_32x32_fill_lanczos_center_3.png">
  <link rel="apple-touch-icon" type="image/png" href="/images/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_192x192_fill_lanczos_center_3.png">

  <link rel="canonical" href="https://svanuytven.github.io/talk/biomina2022/">

  
  
  
  
  
    
    
  
  
  <meta property="twitter:card" content="summary">
  
  <meta property="og:site_name" content="Behind The Bioinformatics">
  <meta property="og:url" content="https://svanuytven.github.io/talk/biomina2022/">
  <meta property="og:title" content="Single-cell genome-plus-transcriptome sequencing without upfront genome preamplification reveals differential cell state plasticity and treatment response between genetic subclones | Behind The Bioinformatics">
  <meta property="og:description" content="Single-cell sequencing techniques allow the study of the subclonal architecture of tumours and reveal the co-occurrence of (driver) mutations as well as their order of acquisition over molecular pseudo-time. The development of multi-omics techniques such as genome-and-transcriptome sequencing (G&amp;T-seq) will enable us to study the diversity of cancer cell states that arises within a tumour, at its most fundamental level, the cell. Recently we developed Gtag&amp;T-seq, an improved G&amp;T-seq method that applies direct genome tagmentation (Gtag), thereby avoiding upfront whole-genome amplification (WGA) and its associated cost and biases. We applied both methodologies to a human melanoma PDX melanoma model before treatment with BRAF and MEK inhibition, and at minimal residual disease. Three genomic subclones were observed at both timepoints, however their relative abundancy at cellular level changes upon drug exposure. Using the RNA of the same single cells, we showed that transcriptome-based DNA copy number inference has limited resolution and accuracy, underlining the importance of affordable multi-omic approaches. In addition, we discerned 10 focal amplifications on chromosomes 13 and 22 of which only genes of the chr22q11.21 focal amplification showed clear gene dosage effects. Using the Gtag&amp;T data, we were able to determine breakpoints at near base-pair resolution, which is not possible with conventional G&amp;T-seq. Finally, we mapped known melanoma resistant cell states on top of the phylogenetic cell lineage tree revealing that some cell states do not occur in all of the subclones. In conclusion, Gtag&amp;T-seq is a novel, low-cost, and accurate single-cell multi-omics method that enables the exploration of somatic genetic alterations and their functional consequences in single cells at scale."><meta property="og:image" content="img/map[gravatar:%!s(bool=false) shape:circle]">
  <meta property="twitter:image" content="img/map[gravatar:%!s(bool=false) shape:circle]"><meta property="og:locale" content="en-us">
  
    
      <meta property="article:published_time" content="2018-10-23T15:53:54&#43;02:00">
    
    <meta property="article:modified_time" content="2018-10-23T15:53:54&#43;02:00">
  

  



  
    




<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "Event",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://svanuytven.github.io/talk/biomina2022/"
  },
  "name": "Single-cell genome-plus-transcriptome sequencing without upfront genome preamplification reveals differential cell state plasticity and treatment response between genetic subclones",
  
  "location": {
    "@type": "Place",
    "name": "Campus Middelheim, Antwerp"
  },
  
  
  "startDate": "2018-10-23T15:53:54+02:00",
  
  
  "performer": {
    "@type": "Person",
    "name": "Sebastiaan Vanuytven"
  },
  
  "description": "Single-cell sequencing techniques allow the study of the subclonal architecture of tumours and reveal the co-occurrence of (driver) mutations as well as their order of acquisition over molecular pseudo-time. The development of multi-omics techniques such as genome-and-transcriptome sequencing (G\u0026T-seq) will enable us to study the diversity of cancer cell states that arises within a tumour, at its most fundamental level, the cell. Recently we developed Gtag\u0026T-seq, an improved G\u0026T-seq method that applies direct genome tagmentation (Gtag), thereby avoiding upfront whole-genome amplification (WGA) and its associated cost and biases. We applied both methodologies to a human melanoma PDX melanoma model before treatment with BRAF and MEK inhibition, and at minimal residual disease. Three genomic subclones were observed at both timepoints, however their relative abundancy at cellular level changes upon drug exposure. Using the RNA of the same single cells, we showed that transcriptome-based DNA copy number inference has limited resolution and accuracy, underlining the importance of affordable multi-omic approaches. In addition, we discerned 10 focal amplifications on chromosomes 13 and 22 of which only genes of the chr22q11.21 focal amplification showed clear gene dosage effects. Using the Gtag\u0026T data, we were able to determine breakpoints at near base-pair resolution, which is not possible with conventional G\u0026T-seq. Finally, we mapped known melanoma resistant cell states on top of the phylogenetic cell lineage tree revealing that some cell states do not occur in all of the subclones. In conclusion, Gtag\u0026T-seq is a novel, low-cost, and accurate single-cell multi-omics method that enables the exploration of somatic genetic alterations and their functional consequences in single cells at scale."
}
</script>

  


  


  





  <title>Single-cell genome-plus-transcriptome sequencing without upfront genome preamplification reveals differential cell state plasticity and treatment response between genetic subclones | Behind The Bioinformatics</title>

</head>

<body id="top" data-spy="scroll" data-offset="70" data-target="#TableOfContents" >

  <aside class="search-results" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" spellcheck="false" type="search">
        
      </div>

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>


  







<nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id="navbar-main">
  <div class="container">

    
    <div class="d-none d-lg-inline-flex">
      <a class="navbar-brand" href="/">Behind The Bioinformatics</a>
    </div>
    

    
    <button type="button" class="navbar-toggler" data-toggle="collapse"
            data-target="#navbar-content" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
    <span><i class="fas fa-bars"></i></span>
    </button>
    

    
    <div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none">
      <a class="navbar-brand" href="/">Behind The Bioinformatics</a>
    </div>
    

    
    
    <div class="navbar-collapse main-menu-item collapse justify-content-start" id="navbar-content">

      
      <ul class="navbar-nav d-md-inline-flex">
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#about"><span>Home</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#experience"><span>Experience</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#publications"><span>Publications</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#posts"><span>Posts</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#talks"><span>Talks</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#accomplishments"><span>Accomplish­ments</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/files/cv.pdf"><span>CV</span></a>
        </li>

        
        

      

        
      </ul>
    </div>

    <ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2">
      
      <li class="nav-item">
        <a class="nav-link js-search" href="#"><i class="fas fa-search" aria-hidden="true"></i></a>
      </li>
      

      
      <li class="nav-item">
        <a class="nav-link js-dark-toggle" href="#"><i class="fas fa-moon" aria-hidden="true"></i></a>
      </li>
      

      

    </ul>

  </div>
</nav>


  <div class="pub">

  












  

  
  
  
<div class="article-container pt-3">
  <h1>Single-cell genome-plus-transcriptome sequencing without upfront genome preamplification reveals differential cell state plasticity and treatment response between genetic subclones</h1>

  

  
    


<div class="article-metadata">

  
  

  

  

  

  
  
  

  
  

</div>

    














  
</div>



  <div class="article-container">

    
      <h3>Abstract</h3>
      <p class="pub-abstract">Single-cell sequencing techniques allow the study of the subclonal architecture of tumours and reveal the co-occurrence of (driver) mutations as well as their order of acquisition over molecular pseudo-time. The development of multi-omics techniques such as genome-and-transcriptome sequencing (G&amp;T-seq) will enable us to study the diversity of cancer cell states that arises within a tumour, at its most fundamental level, the cell. Recently we developed Gtag&amp;T-seq, an improved G&amp;T-seq method that applies direct genome tagmentation (Gtag), thereby avoiding upfront whole-genome amplification (WGA) and its associated cost and biases. We applied both methodologies to a human melanoma PDX melanoma model before treatment with BRAF and MEK inhibition, and at minimal residual disease. Three genomic subclones were observed at both timepoints, however their relative abundancy at cellular level changes upon drug exposure. Using the RNA of the same single cells, we showed that transcriptome-based DNA copy number inference has limited resolution and accuracy, underlining the importance of affordable multi-omic approaches. In addition, we discerned 10 focal amplifications on chromosomes 13 and 22 of which only genes of the chr22q11.21 focal amplification showed clear gene dosage effects. Using the Gtag&amp;T data, we were able to determine breakpoints at near base-pair resolution, which is not possible with conventional G&amp;T-seq. Finally, we mapped known melanoma resistant cell states on top of the phylogenetic cell lineage tree revealing that some cell states do not occur in all of the subclones. In conclusion, Gtag&amp;T-seq is a novel, low-cost, and accurate single-cell multi-omics method that enables the exploration of somatic genetic alterations and their functional consequences in single cells at scale.</p>
    

    <div class="row">
      <div class="col-md-1"></div>
      <div class="col-md-10">
        <div class="row">
          <div class="col-12 col-md-3 pub-row-heading">Date</div>
          <div class="col-12 col-md-9">
            Oct 23, 2018 3:53 PM
          </div>
        </div>
      </div>
      <div class="col-md-1"></div>
    </div>
    <div class="d-md-none space-below"></div>

    
    <div class="row">
      <div class="col-md-1"></div>
      <div class="col-md-10">
        <div class="row">
          <div class="col-12 col-md-3 pub-row-heading">Event</div>
          <div class="col-12 col-md-9">
            <a href="https://www.biomina.be/node/103" target="_blank" rel="noopener">
            Biomina Lunch Talks 2022
            </a>
          </div>
        </div>
      </div>
      <div class="col-md-1"></div>
    </div>
    <div class="d-md-none space-below"></div>
    

    
    <div class="row">
      <div class="col-md-1"></div>
      <div class="col-md-10">
        <div class="row">
          <div class="col-12 col-md-3 pub-row-heading">Location</div>
          <div class="col-12 col-md-9">Campus Middelheim, Antwerp</div>
          
        </div>
      </div>
      <div class="col-md-1"></div>
    </div>
    <div class="d-md-none space-below"></div>
    

    <div class="space-below"></div>

    <div class="article-style">
      
    </div>

    





<div class="article-tags">
  
  <a class="badge badge-light" href="/tags/cancer/">Cancer</a>
  
  <a class="badge badge-light" href="/tags/single-cell/">Single-cell</a>
  
  <a class="badge badge-light" href="/tags/multi-omics/">Multi-omics</a>
  
  <a class="badge badge-light" href="/tags/development/">Development</a>
  
  <a class="badge badge-light" href="/tags/workshop/">Workshop</a>
  
</div>



<div class="share-box" aria-hidden="true">
  <ul class="share">
    
      
      
      
        
      
      
      
      <li>
        <a href="https://twitter.com/intent/tweet?url=https://svanuytven.github.io/talk/biomina2022/&amp;text=Single-cell%20genome-plus-transcriptome%20sequencing%20without%20upfront%20genome%20preamplification%20reveals%20differential%20cell%20state%20plasticity%20and%20treatment%20response%20between%20genetic%20subclones" target="_blank" rel="noopener" class="share-btn-twitter">
          <i class="fab fa-twitter"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="https://www.facebook.com/sharer.php?u=https://svanuytven.github.io/talk/biomina2022/&amp;t=Single-cell%20genome-plus-transcriptome%20sequencing%20without%20upfront%20genome%20preamplification%20reveals%20differential%20cell%20state%20plasticity%20and%20treatment%20response%20between%20genetic%20subclones" target="_blank" rel="noopener" class="share-btn-facebook">
          <i class="fab fa-facebook"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="mailto:?subject=Single-cell%20genome-plus-transcriptome%20sequencing%20without%20upfront%20genome%20preamplification%20reveals%20differential%20cell%20state%20plasticity%20and%20treatment%20response%20between%20genetic%20subclones&amp;body=https://svanuytven.github.io/talk/biomina2022/" target="_blank" rel="noopener" class="share-btn-email">
          <i class="fas fa-envelope"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="https://www.linkedin.com/shareArticle?url=https://svanuytven.github.io/talk/biomina2022/&amp;title=Single-cell%20genome-plus-transcriptome%20sequencing%20without%20upfront%20genome%20preamplification%20reveals%20differential%20cell%20state%20plasticity%20and%20treatment%20response%20between%20genetic%20subclones" target="_blank" rel="noopener" class="share-btn-linkedin">
          <i class="fab fa-linkedin-in"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="https://web.whatsapp.com/send?text=Single-cell%20genome-plus-transcriptome%20sequencing%20without%20upfront%20genome%20preamplification%20reveals%20differential%20cell%20state%20plasticity%20and%20treatment%20response%20between%20genetic%20subclones%20https://svanuytven.github.io/talk/biomina2022/" target="_blank" rel="noopener" class="share-btn-whatsapp">
          <i class="fab fa-whatsapp"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="https://service.weibo.com/share/share.php?url=https://svanuytven.github.io/talk/biomina2022/&amp;title=Single-cell%20genome-plus-transcriptome%20sequencing%20without%20upfront%20genome%20preamplification%20reveals%20differential%20cell%20state%20plasticity%20and%20treatment%20response%20between%20genetic%20subclones" target="_blank" rel="noopener" class="share-btn-weibo">
          <i class="fab fa-weibo"></i>
        </a>
      </li>
    
  </ul>
</div>












  






  
  
  
    
  
  
  
  <div class="media author-card content-widget-hr">
    
      
      <img class="avatar mr-3 avatar-circle" src="/authors/admin/avatar_hu72581610d91fe5871f3fcafe7a3e7ff1_1078029_270x270_fill_q90_lanczos_center.JPG" alt="Avatar">
    

    <div class="media-body">
      <h5 class="card-title"><a href="https://svanuytven.github.io/">Sebastiaan Vanuytven</a></h5>
      <h6 class="card-subtitle">Postdoctoral bioinformatician</h6>
      <p class="card-text">My research interests include Single-Cell (multi-)Omics, oncobiology and (Bayesian) statistics</p>
      <ul class="network-icon" aria-hidden="true">
  
    
    
    
      
    
    
    
    
    
    <li>
      <a href="mailto:Sebastiaan.vanuytven@kuleuven.be" >
        <i class="fas fa-envelope"></i>
      </a>
    </li>
  
    
    
    
      
    
    
    
    
    
      
    
    <li>
      <a href="https://twitter.com/Svanuytven" target="_blank" rel="noopener">
        <i class="fab fa-twitter"></i>
      </a>
    </li>
  
    
    
    
      
    
    
    
    
    
      
    
    <li>
      <a href="https://www.linkedin.com/in/svanuytven/" target="_blank" rel="noopener">
        <i class="fab fa-linkedin"></i>
      </a>
    </li>
  
    
    
    
      
    
    
    
    
    
      
    
    <li>
      <a href="https://github.com/Svanuytven" target="_blank" rel="noopener">
        <i class="fab fa-github"></i>
      </a>
    </li>
  
    
    
    
    
    
    
    
      
    
    <li>
      <a href="https://scholar.google.be/citations?hl=en&amp;user=7eZpsO0AAAAJ" target="_blank" rel="noopener">
        <i class="ai ai-google-scholar"></i>
      </a>
    </li>
  
    
    
    
    
    
    
    
      
    
    <li>
      <a href="http://orcid.org/0000-0001-6723-0132" target="_blank" rel="noopener">
        <i class="ai ai-orcid"></i>
      </a>
    </li>
  
</ul>

    </div>
  </div>









  
  
  <div class="article-widget content-widget-hr">
    <h3>Related</h3>
    <ul>
      
      <li><a href="/talk/crgsymposium/">Single-cell genome-plus-transcriptome sequencing without upfront genome preamplification reveals differential cell state plasticity and treatment response between genetic subclones</a></li>
      
      <li><a href="/talk/trainmalta/">Single-cell multi-omics to detect genetic variants in cancer and development</a></li>
      
      <li><a href="/talk/rsgsymposium2018/">Single-cell multi-omics sequencing to understand the nature, extent and biology of cellular heterogeneity in breast cancer</a></li>
      
      <li><a href="/talk/eshg2018/">Single-cell multi-omics sequencing to understand the nature, extent and biology of cellular heterogeneity in breast cancer</a></li>
      
      <li><a href="/talk/beshg2018/">Single-cell multi-omics sequencing to understand the nature, extent and biology of cellular heterogeneity in breast cancer</a></li>
      
    </ul>
  </div>
  



  </div>
</div>

      

    
    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js" integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js" integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js" integrity="sha256-yt2kYMy0w8AbtF89WXb2P1rfjcP/HTHLT7097U8Y5b8=" crossorigin="anonymous"></script>

      

      
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.10/highlight.min.js" integrity="sha256-1zu+3BnLYV9LdiY85uXMzii3bdrkelyp37e0ZyTAQh0=" crossorigin="anonymous"></script>
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.10/languages/r.min.js"></script>
        
      

    

    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.5.1/leaflet.js" integrity="sha256-EErZamuLefUnbMBQbsEqu1USa+btR2oIlCpBJbyD4/g=" crossorigin="anonymous"></script>
    

    
    
    <script>const code_highlighting = true;</script>
    

    
    
    
    
    
    
    <script>
      const search_config = {"indexURI":"/index.json","minLength":1,"threshold":0.3};
      const i18n = {"no_results":"No results found","placeholder":"Search...","results":"results found"};
      const content_type = {
        'post': "Posts",
        'project': "Projects",
        'publication' : "Publications",
        'talk' : "Talks"
        };
    </script>
    

    
    

    
    
    <script id="search-hit-fuse-template" type="text/x-template">
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script>
    

    
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js" integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js" integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin="anonymous"></script>
    

    
    

    
    

    
    
    
    
    
    
    
    
    
      
    
    
    
    
    <script src="/js/academic.min.946cacb7918c5cc5ead99accf3e34d41.js"></script>

    






  
  
  <div class="container">
    <footer class="site-footer">
  

  <p class="powered-by">
    

    Powered by the
    <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic theme</a> for
    <a href="https://gohugo.io" target="_blank" rel="noopener">Hugo</a>.

    
    <span class="float-right" aria-hidden="true">
      <a href="#" class="back-to-top">
        <span class="button_icon">
          <i class="fas fa-chevron-up fa-2x"></i>
        </span>
      </a>
    </span>
    
  </p>
</footer>

  </div>
  

  
<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <pre><code class="tex hljs"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

</body>
</html>
